# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/glibc/mips.patch.cross0
# Copyright (C) 2015 - 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- glibc-2.36/sysdeps/mips/preconfigure.vanilla	2022-08-02 12:19:10.823984497 +0200
+++ glibc-2.36/sysdeps/mips/preconfigure	2022-08-02 12:20:23.841981586 +0200
@@ -18,7 +18,7 @@
 	elif test "$abiflag" = "_ABI64" ; then
 		machine=mips/mips64/n64
 	else
-		as_fn_error 1 "Unable to determine ABI." "$LINENO" 5
+		machine=mips/mips64/n64
 	fi
 	# $config_machine is not really needed here but the slash after
 	# $machine is needed by the case statement in
